### PR TITLE
Add Jellyfin documentation

### PR DESCRIPTION
**Pull request by RunC0deRun**
## Proposed change

Add the documentation for the newly proposed integration of the [Jellyfin](https://jellyfin.org/) media system.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/44401
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2099

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/jellyfin.markdown [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+The `jellyfin` integration exposes a [Jellyfin](https://jellyfin.org/) server as a Media Source in Home Assistant.
+Support is currently limited to music libraries only. Other libraries will not appear in the Media Browser. This integration has been tested with Jellyfin server version 10.6.4, but should support older versions as well.
+
+## Configuration
+
+The Jellyfin integration is configured through the Home Assistant user interface. In the Configuration page, select Integrations and click on the `+` button and choose Jellyfin.
+
+{% configuration %}
```
Comment (frenck) [inline source/_integrations/jellyfin.markdown]: This configuration block is used for YAML configuration. If you want to supply such an index, please use the basic configuration block instead (examples can be found in the Lovelace card documentation).

File: source/_integrations/jellyfin.markdown [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  - Media Source
+ha_release: '2021.1'
+ha_iot_class: Local Polling
+ha_codeowners:
+  - '@j-stienstra'
+ha_config_flow: true
+ha_domain: jellyfin
+---
+
+The `jellyfin` integration exposes a [Jellyfin](https://jellyfin.org/) server as a Media Source in Home Assistant.
```
Comment (frenck) [inline source/_integrations/jellyfin.markdown]:
```suggestion
The Jellyfin integration exposes a [Jellyfin](https://jellyfin.org/) server as a Media Source in Home Assistant.
```

File: source/_integrations/jellyfin.markdown [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  description: The password of the supplied user.
+  required: true
+  type: string
+{% endconfiguration %}
+
+## Jellyfin server configuration
+
+The Jellyfin integration retrieves media items from your Jellyfin libraries using a Artist -> Album -> Track hierarchy. In order for the Media Browser to display thumbnails for Artists, Jellyfin has to include metadata for artists and albums. This is not enabled by default. 
+
+To enable this navigate to the Jellyfin dashboard and select Plugins. Choose which provider you would like to use for metadata retrieval (AudioDB and MusicBrainz are the two default providers) and select Settings. Enable the checkbox for "Enable this provider for metadata searches on artists and albums.".
```
Comment (frenck) [inline source/_integrations/jellyfin.markdown]:
```suggestion
## Jellyfin server configuration

The Jellyfin integration retrieves media items from your Jellyfin libraries using an Artist -> Album -> Track hierarchy. In order for the Media Browser to display thumbnails for Artists, Jellyfin has to include metadata for artists and albums. This is not enabled by default. 

To enable this to navigate to the Jellyfin dashboard and select Plugins. Choose which provider you would like to use for metadata retrieval (AudioDB and MusicBrainz are the two default providers) and select Settings. Enable the checkbox for "Enable this provider for metadata searches on artists and albums.".
```

**Review by frenck — CHANGES_REQUESTED** [review]
Some small changes requested. Could you take a look?

Thanks already! 👍

File: source/_integrations/jellyfin.markdown (hunk 1 above)
Comment (RunC0deRun) [reply source/_integrations/jellyfin.markdown]: Done

File: source/_integrations/jellyfin.markdown (hunk 2 above)
Comment (RunC0deRun) [reply source/_integrations/jellyfin.markdown]: Done

File: source/_integrations/jellyfin.markdown (hunk 3 above)
Comment (RunC0deRun) [reply source/_integrations/jellyfin.markdown]: I changed 'a' into 'an', but I don't think the extra 'to' needs to be there. I added a comma instead.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/jellyfin.markdown [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ha_config_flow: true
+ha_domain: jellyfin
+---
+
+The Jellyfin integration exposes a [Jellyfin](https://jellyfin.org/) server as a Media Source in Home Assistant.
+Support is currently limited to music libraries only. Other libraries will not appear in the Media Browser. This integration has been tested with Jellyfin server version 10.6.4, but should support older versions as well.
+
+## Configuration
+
+The Jellyfin integration is configured through the Home Assistant user interface. In the Configuration page, select Integrations and click on the `+` button and choose Jellyfin.
```
Comment (frenck) [inline source/_integrations/jellyfin.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

File: source/_integrations/jellyfin.markdown [hunk 5]
```diff
@@ -0,0 +1,32 @@
+---
+title: Jellyfin
+description: Instructions on how to integrate the Jellyfin integration into Home Assistant.
+ha_category:
+  - Media Source
+ha_release: '2021.1'
```
Comment (frenck) [inline source/_integrations/jellyfin.markdown]:
```suggestion
ha_release: '2021.12'
```

**Review by frenck — APPROVED** [review]

**Comment by RunC0deRun** [comment]
Thanks @frenck for updating this PR! I saw the parent got merged, but did not get round to updating this one over the weekend.

**Pull request merged**